src/main/java/seedu/address/model/recipe/Ingredient.java

```diff
@@ -81,7 +81,7 @@ public static boolean isValidQuantity(double test) {
 
     @Override
     public String toString() {
-        return String.format("%s: %s %s", ingredientName, quantity, quantifier);
+        return String.format("%s: %s %s", ingredientName, quantity, quantifier).trim();
```

Comment:
don't think this line is necessary imo since the given string format doesn't have leading/trailing spaces, so the trim() method doesn't really do anything here
```suggestion
        return String.format("%s: %s %s", ingredientName, quantity, quantifier);
```